Jenkinsfile

```diff
@@ -268,6 +268,14 @@ def withBeatsEnv(Map args = [:], Closure body) {
             git config --global user.email "beatsmachine@users.noreply.github.com"
             git config --global user.name "beatsmachine"
           fi''')
+
+        // Add more stability when dependencies are not accessible temporarily
+        // See https://github.com/elastic/beats/issues/21609
+        // retry/try/catch approach reports errors, let's avoid it to keep the
+        // notifications cleaner.
+        if (sh(label: 'Fetch go dependencies', script: 'go get ./...', returnStatus: true) > 0) {
+          sh(label: 'Fetch go dependencies - retry', script: 'go get ./...', returnStatus: true)
```

Comment:
Do we need the return status here?

Reply:
Yes, this will help to move forward even if the fetching dependencies step failed. This is to add more resilience if the go proxy repo is down temporarily. 
The `body()` will be executed anyway and if some dependencies are missing it should be in charge to download them.